languages/ruby/exercises/concept/strings/.docs/introduction.md

```diff
@@ -0,0 +1,4 @@
+# Introduction
+
+A `String` in Ruby is an object that holds and manipulates an arbitrary sequence of bytes, typically representing characters.
```

Comment:
- I think we should keep "in Ruby"
- I agree with referencing the documentation
- Can we name the links rather than referencing them by number pls?

Reply:
Just something to consider: adding the "is an object" bit is probably an important Ruby detail, but it does require the student to know what an object means in Ruby.

Reply:
Personally, I feel the documentation reference is not optimal. It is such a technical document that people new to Ruby could easily get stuck there I feel (I know I had a profound distaste of the Ruby doc pages when I learnt Ruby). I'd suggest either a more basic document, or only referencing the documentation in the `hints.md` file.

Reply:
We could go with something like "A string in Ruby is an arbitrary sequence of bytes, typically representing characters." with no links so the student only needs to know about `bytes` and `characters` What do you think?

Reply:
I really like that! @iHiD?

Reply:
> Personally, I feel the documentation reference is not optimal.

I agree with this. Maybe we just leave that link out.

> adding the "is an object" bit is probably an important Ruby detail, but it does require the student to know what an object means in Ruby.

I think it's really important in Ruby to say it's an object. If someone doesn't know what that means, they can google it, or they can ignore it, but I think it's an important piece of information for people who don't know, because it determines how you think about interacting with Strings. 

I think something like this is fine, personally:

> A `String` in Ruby is an object that holds and manipulates an arbitrary sequence of bytes, typically representing characters. Strings are manipulated by calling the string's methods.

Reply:
Agree objects are a key concept on Ruby, I like this last version 👍

Reply:
Me too